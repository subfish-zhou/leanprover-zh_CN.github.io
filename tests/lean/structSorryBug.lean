instance has_arr : HasArr Preorder := { Arr := Function }

def foo : Nat := { first := 10 }
