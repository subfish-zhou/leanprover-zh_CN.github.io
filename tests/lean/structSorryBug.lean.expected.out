structSorryBug.lean:1:19-1:25: error: unknown identifier 'HasArr'
structSorryBug.lean:3:17-3:32: error: invalid {...} notation, structure type expected
  Nat
